packages/account/src/utils/Types.ts

```diff
@@ -49,3 +49,15 @@ export type InitilizationData = {
   accountIndex?: number
   signerAddress?: string
 }
+
+export type EstimateUserOpGasDto = {
+  userOp: Partial<UserOperation>
+  overrides?: Overrides
+  skipBundlerGasEstimation?: boolean
+}
+
+export type BuildUserOpDto = {
```

Comment:
this is tricky because now they will have to pass buildUserOp({transactions: [tx1, tx2, tx3]}

Reply:
yes, if we need to go with DTO then its mandatory